package/README.md

```diff
@@ -0,0 +1,59 @@
+# Short Term History (STH) package
+
+This directory contains tools to make and distribute STH in RPM format used, mainly, in Red Hat, CentOS and
+Fedora Linux distributions.
+
+With the RPM is included:
```

Comment:
I've seen this was fixed in https://github.com/telefonicaid/IoT-STH/commit/1b710efe40b04c6944ce7a18de42fa3ebcc8807d